cap-ci/cap-nightly-caasp.yaml

```diff
@@ -0,0 +1,67 @@
+workertags:
+  caasp4: suse-internal
+
+jobs:
+  deploy-k8s: false
+  deploy-kubecf: true
+  smoke-tests: true
+  cf-acceptance-tests-brain: true
+  sync-integration-tests: true
+  minibroker-integration-tests: true
+  cf-acceptance-tests: true
+  upgrade-kubecf: true
+  deploy-stratos: false
+  clean-cap: true
+  destroy-kubecf: false
+jobs_ordered:
+- deploy-k8s
+- deploy-kubecf
+- smoke-tests
+- sync-integration-tests
+- cf-acceptance-tests
+- upgrade-kubecf
+- minibroker-integration-tests
+- cf-acceptance-tests-brain
+- deploy-stratos
+- clean-cap
+- destroy-kubecf
+backends:
+  caasp4: true
+  aks: false
+  gke: false
+  eks: false
+availabilities:
+  sa: true
+  ha: true
+  all: true
+schedulers:
+  diego: true
+  eirini: true
+catapult:
+  uri: https://github.com/SUSE/catapult
+  branch: master
+brain:
+  verbose: false
+  inorder: false
+  include: ""
+  exclude: ""
+cats:
+  nodes: 3
+  flake-attempts: 5
```

Comment:
This sounds like A LOT of attempts. Does it make sense to bump the `timeout-scale` to a way larger number than setting this huge attempt count on flake? Have you tried this before?

Reply:
I think it was tested when this was first introduced to CI / catapult but it's been months, so I don't remember. May be a good thing to bring up in CI meeting as other might remember, or may be a new JIRA card.
Should not be part of this PR.

Reply:
Sounds good. If you could bring that to the meeting, it would be great.